### ISSUE TITLE
New Account Bundle Purchase: Clicking on Playlist Title Displays Incorrect Title and Songs Listed 

### ISSUE THREAD

**Issue by Mel01000001** [riverscuomo/public-bug-hunt#79]
**Describe the bug**
After creating an account, purchasing a new demo bundle results in the incorrect playlist title and songs being listed, when clicking on the 'Best of Playlists' in Your Library.

**To Reproduce**

Log out of android app
Go to 'Profile' tab
Click on 'No account? Sign up'
Complete sign up process
Go to Market and purchase demo bundle
Go to Your Library and click on 'Best of' bundle playlist that you purchased as well as any other playlists.

_Workaround:  the issue was rectified by closing then reopening the app._

**Expected behavior**
Bundle purchased and all other playlists will display correct title and songs when clicked on.

**Screenshots**
In all the screen recordings, you can see the results after purchasing a demo bundle.  None match the original bundles purchased.

**Smartphone (please complete the following information):**

Device: [Samsung Galaxy S23+]
OS: [Android 13]
Browser [Android app]
Version 1.10.1(181)

**Additional context**

https://github.com/riverscuomo/public-bug-hunt/assets/144282154/06c8d367-f292-4141-af8b-ad3c0954742a

https://github.com/riverscuomo/public-bug-hunt/assets/144282154/6a2f686f-2e5f-427c-93df-6010b3ff013c

https://github.com/riverscuomo/public-bug-hunt/assets/144282154/b8d8d277-df04-496b-b53b-acc2520b4153

**Comment by Mel01000001** [comment]
This continues on 182 (1.10.1).

It didn't happen after the first bundle purchase, but did after the second. 

Again, rectified by closing and reopening the app.

/

**Comment by riverscuomo** [comment]
i think this was related to a null check error here in Track model after purchase
imageFilename: imageFilename ?? this.imageFilename,

**Comment by Mel01000001** [comment]
Issue continues in build 183 (1.10.2), not after first bundle purchase, but after second and rectified by closing and reopening app.

**Comment by Mel01000001** [comment]
Issue continues in build 184 (1.11.0), occurred after first bundle purchase and rectified by closing and reopening app.

**Comment by KCreek1** [comment]
I had this issue briefly on iOS app 185 (1.11.0) on the first new acct I created on my iPhone. Same result as above. As soon as I closed app and reopened then all the playlists functioned normally. 

I created a second new acct on iPhone. The playlists had no issue whatsoever. 

I also created a new acct on iPad. The playlists had no issue whatsoever. 

iPhone SE 2nd gen
iPad 7th gen
iOS 17.0.3
Weezify 185 (1.11.0)
